src/utils/noop.ts

```diff
@@ -0,0 +1,3 @@
+export default function noop() {
```

Comment:
why null instead of undefined or void?
I would expect this to return void:

```
export default function noop(): void {
  // Intentionally left empty.
}
```

Reply:
If it needs to be `null` to satisfy typing, just leave that as a doc-comment:

```
/** No-op function. Returns `null` to satisfy most React typings. */
```

In the future maybe there can be other noops that return `undefined` or `void`, but for right now this looks great.